src/components/atoms/LinkButton/LinkButton.scss

```diff
@@ -0,0 +1,14 @@
+@import "scss/constants.scss";
+
+.button-link {
+  text-align: center;
+  text-decoration: none;
+  margin-top: 20px;
+  background-color: $lighter-intermediate-grey;
+  padding: 12px 16px;
+  border-radius: 28px;
```

Comment:
margins, paddings, border-radiuses should be imported from `constants`

Reply:
https://github.com/sparkletown/sparkle/pull/1396/commits/042a93353ab9ea1d6ea606ef5b7090d8499ff5f7